app/lib/services/voip/MediaSessionInstance.test.ts

```
@@ -1,7 +1,8 @@
 import type { IClientMediaCall } from '@rocket.chat/media-signaling';
-import RNCallKeep from 'react-native-callkeep';
 import { waitFor } from '@testing-library/react-native';
 
+import { voipNative, type InMemoryVoipNative } from './VoipNative';
```

ESLint

[error] There should be no empty line within import group

(import/order)